core/http-auth/src/main/java/software/amazon/awssdk/http/auth/NoAuthAuthScheme.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package software.amazon.awssdk.http.auth;
+
+import software.amazon.awssdk.annotations.SdkPublicApi;
+
+/**
+ * No-auth auth scheme ID.
```

Comment:
```suggestion
 * An auth scheme trait that represent no authentication.
```

Reply:
This is not a *trait* but rather a function of a synthetic trait and that's why I avoided that wording, even more so if we change it to implement an `AuthScheme`.

Reply:
Sorry, didn't mean to say `trait` in there. I suggest,`An auth scheme that represent no authentication.`